Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -59,9 +61,12 @@ xcodebuild -configuration $CONFIGURATION -workspace $NATIVEDIR/GVFS.Native.Mac.x
 USERNOTIFICATIONDIR=$VFS_SRCDIR/GVFS/GVFS.Notifications/VFSForGit.Mac
 USERNOTIFICATIONPROJECT="$USERNOTIFICATIONDIR/VFSForGit.xcodeproj"
 USERNOTIFICATIONSCHEME="VFS For Git"
-updateAppVersionCmd="(cd \"$USERNOTIFICATIONDIR\" && /usr/bin/xcrun agvtool new-marketing-version \"$VERSION\")"
-echo $updateAppVersionCmd
-eval $updateAppVersionCmd
+# don't update version number for developer builds
```

Comment:
Should this comment be up where we set this variable?

Reply:
that would be more appropriate, updated